pkg/cmd/controller/interfaces.go

```diff
@@ -99,7 +107,7 @@ func NewControllerContext(
 
 		ClientBuilder: OpenshiftControllerClientBuilder{
 			ControllerClientBuilder: controller.SAControllerClientBuilder{
-				ClientConfig:         rest.AnonymousClientConfig(clientConfig),
+				ClientConfig:         rest.AnonymousClientConfig(kubeClientConfig),
```

Comment:
@smarterclayton from what I can gather, this client config is used when the controllers call `ctx.ClientBuilder.ClientOrDie(...)`. Every controller in OCM calls this.

My fear is that by using a higher QPS across all controllers, we can DoS the kube apiserver.

Reply:
cc @deads2k @mfojtik